payments/using-golem-on-mainnet/backing-up-your-golem-wallet.md

```diff
@@ -0,0 +1,114 @@
+---
+description: ensuring safety of your Golem funds.
+---
+
+
+# Backing up your Golem wallet
+
+As you're probably aware, in the Ethereum universe, your funds are only as secure as the private key for the account that holds them. Because of that, to ensure that you don't lose access to your GLM/ETH tokens stored on your Golem account, it's crucial for you to be able to back-up your Golem wallet and store the key in a safe storage, separate from the node itself.
+
+## Simple, unsafe export
+
+To create a backup of your Golem wallet, export the keystore with:
```

Comment:
```suggestion
To create a backup of your Golem wallet \(in Ethereum wallet v3 format\), export the keystore with:
```

Reply:
hmm, I wouldn't put this detail here ... unless we give people options to export the keys in any other formats (and the one yagna is using is anyway I think the common one), it doesn't add much and can confuse people that indeed, there is an option to choose a format somewhere